backoffice/pages/catalog/products/[id]/productAttributes.tsx

```diff
@@ -45,7 +45,7 @@ const ProductAttributes: NextPage = () => {
 
   useEffect(() => {
     if (id) {
-      let checkIdValid: boolean;
+      let checkIdValid = true;
```

Comment:
![image](https://user-images.githubusercontent.com/43461631/223352876-843019cf-0b9d-4177-bb5d-3185e79d9ab5.png)

in this image, when we edit a product, go to the tab "Product Attributes", we would see a dropdown to select some attributes for our product, this dropdown will contain a list of attributes that are not attached to the product yet. And the section of code I edited in this pr is responsible for filtering out the attributes attached to the product. As you can see, `checkIdValid` will tell if the attribute is attached to the product or not so that we can decide if we should add it to the dropdown `if(checkId![image](https://user-images.githubusercontent.com/43461631/223352876-843019cf-0b9d-4177-bb5d-3185e79d9ab5.png)

in this image, when we edit a product, go to the tab "Product Attributes", we would see a dropdown to select some attributes for our product, this dropdown will contain a list of attributes that are not attached to the product yet. And the section of code I edited in this pr is responsible for filtering out the attributes attached to the product. As you can see, `checkIdValid` will tell if the attribute is attached to the product or not so that we can decide if we should add it to the dropdown `if (checkIdValid) array.push(obj1); `. So as you can see here, initially `checkIdValid` is not true, so there will be a case in which an attribute not attached to the product might not be added to the dropdown since `checkIdValid` is already false from the beginning.